client/header/activity-panel/panels/orders.js

```diff
@@ -156,7 +164,13 @@ class OrdersPanel extends Component {
 								/>
 							) : null,
 							customerLink: customerUrl ? (
-								<Link href={ customerUrl } type="wc-admin" />
+								<Link
+									href={ customerUrl }
+									onClick={ () =>
+										this.recordOrderEvent( 'customer_name' )
```

Comment:
I know this is what the issue requested, but it seems like it should be prefixed with `order_` to avoid ambiguity.